EventsExpress/ClientApp/src/services/EventsExpressService.js

```diff
@@ -20,6 +20,19 @@ export default class EventsExpressService {
         return res;
     }
 
+    getPhoto = async (url, imgTagId) => {
+        const call = _url => fetch(this._baseUrl + url);
+
+        let res = await call(url);
+
+        if(res.ok){
+            let blob = await res.blob();
+            if(document.getElementById(imgTagId) !== null){
```

Comment:
This is not very obvious and it controls DOM outside of shadow DOM, which is not good. Can we return blob from this method and create a custom img component to apply it properly? Also, does the binding not work?

Reply:
It just doesn't wait for the result :(